src/lib/main.js

```diff
@@ -244,5 +199,58 @@ module.exports = async (argv, env) => {
   }
 };
 
+function buildActions(cmdArgs, skipValidate) {
```

Comment:
Awww you ruined the great big run-on function that we had going with your smaller, well-named, easy-to-read function that encapsulates a distinct subset of the functionality....  :laughing:  :heart: